src/main/java/org/hmhb/program/published/PublishedProgramController.java

```diff
@@ -18,41 +18,41 @@
 import static java.util.Objects.requireNonNull;
 
 @RestController
-public class UserController {
+public class PublishedProgramController {
```

Comment:
This is registering as a rename because all of our REST controllers are almost identical so far.

UserController was removed.  PublishedProgramController and RequestedProgramController were created.

They have almost the same data and endpoints.  RequestedProgram has an extra "accept" endpoint (that I should maybe rename to "publish").